src/app/static/src/scss/partials/form.scss

```diff
@@ -0,0 +1,88 @@
+.dynamic-form {
```

Comment:
These style rules are mostly targeted at the select/multiselect, to make their styles consistent (and consistently Bootstrap-y). I've defined all the styles here to apply only within the dynamic form but actually probably make sense to extend them to the whole app though?

Reply:
Almost certainly! Probably worth doing a style audit at some point to make sure that things like the forgot password form etc aren't getting left behind.

Reply:
Good idea, added a ticket https://vimc.myjetbrains.com/youtrack/issue/mrc-603